src/ms3/instrument_defaults.csv

```diff
@@ -1,652 +1,583 @@
+tarogato,tarogato,,69,pitched,stdNormal,,Tarogato,Tar.,Tarogato,wind.reed.taroagato,Tarogato,,
+octavin,octavin,,69,pitched,stdNormal,,Octavin,Oct.,Octavin,wind.reed.octavin,Octavin,,
+heckelphone-clarinet,heckelphone-clarinet,,68,pitched,stdNormal,G8vb,Heckelphone-clarinet,Hph.-cl.,Heckelphone-clarinet,wind.reed.heckelphone-clarinet,Heckelphone-clarinet,,G8vb
+sopranino shawm,sopranino-shawm,,67,pitched,stdNormal,,Sopranino Shawm,Si. Sh.,Sopranino Shawm,wind.reed.shawm,Sopranino Shawm,,
+soprano shawm,soprano-shawm,,67,pitched,stdNormal,,Soprano Shawm,S. Sh.,Soprano Shawm,wind.reed.shawm,Soprano Shawm,,
+alto shawm,alto-shawm,,67,pitched,stdNormal,,Alto Shawm,A. Sh.,Alto Shawm,wind.reed.shawm,Alto Shawm,,
+tenor shawm,tenor-shawm,,67,pitched,stdNormal,G8vb,Tenor Shawm,T. Sh.,Tenor Shawm,wind.reed.shawm,Tenor Shawm,,G8vb
+bass shawm,bass-shawm,,67,pitched,stdNormal,F,Bass Shawm,B. Sh.,Bass Shawm,wind.reed.shawm,Bass Shawm,,F
+great bass shawm,great-bass-shawm,,67,pitched,stdNormal,G8vb,Great Bass Shawm,G.B. Sh.,Great Bass Shawm,wind.reed.shawm,Great Bass Shawm,,G8vb
+cromorne,cromorne,,67,pitched,stdNormal,,Cromorne,Cr.,Cromorne,wind.reed.cromorne,Cromorne,,
+crumhorn,crumhorn,,67,pitched,stdNormal,,Crumhorn,Crh.,Crumhorn,wind.reed.crumhorn,Crumhorn,,
+soprano crumhorn,soprano-crumhorn,,67,pitched,stdNormal,,Soprano Crumhorn,S. Crh.,Soprano Crumhorn,wind.reed.crumhorn.soprano,Soprano Crumhorn,,
+alto crumhorn,alto-crumhorn,,67,pitched,stdNormal,,Alto Crumhorn,A. Crh.,Alto Crumhorn,wind.reed.crumhorn.alto,Alto Crumhorn,,
+tenor crumhorn,tenor-crumhorn,,67,pitched,stdNormal,G8vb,Tenor Crumhorn,T. Crh.,Tenor Crumhorn,wind.reed.crumhorn.tenor,Tenor Crumhorn,,G8vb
+bass crumhorn,bass-crumhorn,,67,pitched,stdNormal,F,Bass Crumhorn,B. Crh.,Bass Crumhorn,wind.reed.crumhorn.bass,Bass Crumhorn,,F
+greatbass crumhorn,greatbass-crumhorn,,67,pitched,stdNormal,F,Greatbass Crumhorn,Gb. Crh.,Greatbass Crumhorn,wind.reed.crumhorn.great-bass,Greatbass Crumhorn,,F
+cornamuse,cornamuse,,67,pitched,stdNormal,,Cornamuse,Cm.,Cornamuse,wind.reed.cornamuse,Cornamuse,,
+soprano cornamuse,soprano-cornamuse,,67,pitched,stdNormal,,Soprano Cornamuse,S. Cm.,Soprano Cornamuse,wind.reed.cornamuse,Soprano Cornamuse,,
+alto cornamuse,alto-cornamuse,,67,pitched,stdNormal,,Alto Cornamuse,A. Cm.,Alto Cornamuse,wind.reed.cornamuse,Alto Cornamuse,,
+tenor cornamuse,tenor-cornamuse,,67,pitched,stdNormal,G8vb,Tenor Cornamuse,T. Cm.,Tenor Cornamuse,wind.reed.cornamuse,Tenor Cornamuse,,G8vb
+bass cornamuse,bass-cornamuse,,67,pitched,stdNormal,F,Bass Cornamuse,B. Cm.,Bass Cornamuse,wind.reed.cornamuse,Bass Cornamuse,,F
+f duduk,f-duduk,,71,pitched,stdNormal,G8va,F Duduk,F Du.,F Duduk,wind.reed.duduk,F Duduk,,G8va
+e duduk,e-duduk,,71,pitched,stdNormal,G8va,E Duduk,E Du.,E Duduk,wind.reed.duduk,E Duduk,,G8va
+duduk,d-duduk,,71,pitched,stdNormal,G8va,Duduk 2,Du. 2,Duduk,wind.reed.duduk,Duduk 2,,G8va
```

Comment:
`celo` =` Čelo`, not `Violoncello`. There is no `Cello` when I create` instrument_defaults.csv` based on the `mscx` file.

Reply:
I'm sorry for the format of warnings, I will change them. How should I describe then that there is no values to update for example for `violinI` because it is not recognised,  so I have to normalise them?

Reply:
> 

That wasn't done on purpose but automatically. I just get fields of the `mscx` file containing all instruments

Reply:
> That wasn't done on purpose but automatically. I just get fields of the mscx file containing all instruments.

OK, then have you changed this `mscx` file containing all the instruments, or the code that extracts them? Because there is a large number of changes here... Please [inspect and study them](https://github.com/johentsch/ms3/commit/1100190d83491b16ffb9ea70d0976634c4d4c94e#diff-390d7d551c04c9142be4490006303a6336e41203edd7ccd501dd9b3add655371) carefully and check (for example) why
* before there were two lines `(104) duduk,duduk, ...` and `(107) duduk.1,d-duduk, ...` and
* now there is only one line `(106) duduk,d-duduk, ...`